packages/clerk-js/src/ui/components/UserButton/useMultisessionActions.tsx

```diff
@@ -12,8 +12,7 @@ type UseMultisessionActionsParams = {
   actionCompleteCallback?: () => void;
   navigateAfterSignOut?: () => any;
   navigateAfterMultiSessionSingleSignOut?: () => any;
-  navigateAfterSwitchSession?: () => any;
-  afterSignInUrl?: string;
+  afterSwitchSession?: string;
```

Comment:
nit: to be consistent with other `afterXXUrl` properties

```suggestion
  afterSwitchSessionUrl?: string;
```

Reply:
thanks for identifying this one